### PR TITLE
PR for Solved Comments and Test cases

### PR DESCRIPTION
**Pull request by alpit4**

### CODE REVIEW

File: src/Patient_Dashboard/UploadDocumentPage/FileTypeSection.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          onChange={(value) => setSelectedFileType(value)}
+        >
+          <RadioButton
+            value="Prescription"
+            labelText="Prescription"
+            className="radio-button"
+          />
+          <RadioButton
+            value="Referral Documents"
+            labelText="Referral Documents"
```
Comment (gsluthra) [inline src/Patient_Dashboard/UploadDocumentPage/FileTypeSection.js:24]: These options come from Configuration. I think it is OK to hardcode in this PR. In future JIRA tickets we will have to make this come from configuration.

Comment (alpit4) [reply src/Patient_Dashboard/UploadDocumentPage/FileTypeSection.js:24]: I will raise the PR for this differently.

**Comment by gsluthra** [comment]
There is a PENDING issue that on upload of documents using this code, the docs are not visible inside Bahmni UI. That needs to be debugged. Will approve/merge this PR for now. Since its already gotten too big.

**Review by gsluthra — APPROVED** [review]

**Comment by alpit4** [comment]
I will raise a different PR for the showing of the Uploaded Documents on the clinical UI.

**Pull request closed without merging**